### ISSUE TITLE
"OSError: [Errno 104] ECONNRESET" Error

### ISSUE THREAD

**Issue by robtech21** [fabiopolancoe/ESPython-DUCO-Miner#1]
So I setup my miner today and it keeps throwing the following error:

`OSError: [Errno 104] ECONNRESET`

Is there something I need to change with the router or what?

**Comment by nicolaskn** [comment]
I have the same issue as well. Looks like the connection is dropping between the send/receive of a found hash. 
I believe the issue is the server receiving the data and it maybe terminating the connection since the received data back is "None".

Planning to fork the code tonight or this weekend to look into it more. We may need to add a wait until buffer is complete.

**Comment by robtech21** [comment]
> I have the same issue as well. Looks like the connection is dropping between the send/receive of a found hash.
> I believe the issue is the server receiving the data and it maybe terminating the connection since the received data back is "None".
> 
> Planning to fork the code tonight or this weekend to look into it more. We may need to add a wait until buffer is complete.

I tried increasing the timeout delay, but it didn't work. Maybe you could figure something out.

**Comment by fabiopolancoe** [comment]
Yeah, the miner's outdated, I'll try to update it later, I haven't been working on it.

**Comment by fabiopolancoe** [comment]
Sorry for the inconvenience.
